_data/kulgarima/kulgarima.md

```diff
@@ -0,0 +1,7 @@
+---
+name: KUL GARIMA # No longer than 28 characters
+institution: JSS INSTITUTE OF TECHNICAL EDUCATION, NOIDA, INDIA 🚩 # no longer than 58 characters
```

Comment:
Please do not include special characters

Reply:
I made the changes do I have to fill the shipment form again?